src/definitions/modules/calendar.js

```diff
@@ -758,13 +777,13 @@
                         var ms1d = 864e5, // milliseconds in a day
                             ms7d = 7 * ms1d; // milliseconds in a week
 
-                        return function () { // return a closure so constants get calculated only once
+                        return (function () { // return a closure so constants get calculated only once
```

Comment:
the closure is invoked immediately and not returned, is it really useful?

Reply:
> the closure is not returned (invoked immediately)

I think it is returned (wouldn't work totherwise)
```javascript
return function () { // return a closure so constants get calculated only once
...
}();
```
>  is it really useful?

tbh, i just took that calculation function from the credited website back then 😆

Reply:
`let x = (function () {})( return 5; );` (function evaluated immediatelly) is basically a `let x = 5;`

maybe it has something to do with https://eslint.org/docs/latest/rules/wrap-iife

still, I think "function evaluated immediatelly" is not needed here